post_installation_configuration/coreos-layering.adoc

```diff
@@ -82,19 +104,41 @@ Because libreswan requires additional RHEL packages, the image must be built on
 ** SSH Key management packages.
 --
 +
-.Example Containerfile to apply a third-party package from EPEL
+.Example on-cluster Containerfile to apply a third-party package from EPEL
+[source,yaml]
+----
+FROM configs AS final
+
+#Enable EPEL (more info at https://docs.fedoraproject.org/en-US/epel/ ) and install htop
+RUN rpm-ostree install https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
+    rpm-ostree install htop && \
+    ostree container commit
+----
++
+.Example out-of-cluster Containerfile to apply a third-party package from EPEL
 [source,yaml]
 ----
 include::https://raw.githubusercontent.com/openshift/rhcos-image-layering-examples/master/htop/Containerfile[]
 ----
 +
-.Example Containerfile to apply a third-party package that has {op-system-base} dependencies
+This Containerfile installs the Linux fish program. Because fish requires additional RHEL packages, the image must be built on an entitled {op-system-base} host. For RHEL entitlements to work, you must copy the `etc-pki-entitlement` secret into the `openshift-machine-api` namespace. 
```

Comment:
```suggestion
This Containerfile installs the Linux `fish` program. Because `fish` requires additional RHEL packages, the image must be built on an entitled {op-system-base} host. For RHEL entitlements to work, you must copy the `etc-pki-entitlement` secret into the `openshift-machine-api` namespace. 
```
?